scripts/openwrt_upload.sh

```diff
@@ -1,8 +1,11 @@
 #!/usr/bin/env bash
 set -eux
-export TARGET=mipsel
-export TRIPLE=mipsel-unknown-linux-musl
+export TARGET=mips
+export TRIPLE=mips-unknown-linux-musl
 export ROUTER_IP=192.168.10.1
 bash scripts/openwrt_build_$TARGET.sh
+set +e
+ssh root@$ROUTER_IP killall -9 rita
```

Comment:
Why not just `|| exit 1`?

Reply:
we don't want to exit, we specifically want to continue.

Reply:
True, confused the sign